src/FileUpload.jsx

```diff
@@ -77,12 +80,14 @@ export default function FileUpload() {
       const response = await axios.get("http://localhost:5000/auth/status", {
         withCredentials: true,
       });
+      setCheckingAuth(false);
```

Comment:
Make sure the state variable names match if the other is changed.
```suggestion
      setCheckingAuthorization(false);
```